CLAUDE.md

```
@@ -7,636 +7,250 @@
-
-### 4. Platform Constants
-
-**Reason:** Constant may not exist on all platforms
-
-```python
-# ✅ CORRECT - os.O_NOFOLLOW doesn't exist on Windows
-if hasattr(os, "O_NOFOLLOW"):
-    flags |= os.O_NOFOLLOW
-```
+1. **Destructors (`__del__`)** - Can be called during failed initialization
+   ```python
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)